sbncode/Calibration/TrackCaloSkimmer.h

```diff
@@ -161,7 +162,7 @@ class sbn::TrackCaloSkimmer : public art::EDAnalyzer {
     const std::vector<art::Ptr<simb::MCParticle>> &mcparticles,
     const std::vector<geo::BoxBoundedGeo> &active_volumes,
     const std::vector<std::vector<geo::BoxBoundedGeo>> &tpc_volumes,
-    const std::map<int, std::vector<std::pair<geo::WireID, const sim::IDE*>>> id_to_ide_map,
+    const std::map<int, std::vector<caf::ParticleIDE>> id_to_ide_map,
```

Comment:
Object parameters should be passed by reference unless there are explicit reasons to do otherwise ([`[CF-112]`](https://sbnsoftware.github.io/sbn/codingconv/CodingConventionsExplained.html#CF112):
```suggestion
    const std::map<int, std::vector<caf::ParticleIDE>>& id_to_ide_map,
```